watertap/examples/flowsheets/case_studies/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -658,6 +678,184 @@ def solve(m, solver=None):
     return results
 
 
+def add_costing(m):
+    m.fs.costing = WaterTAPCosting()
+    m.fs.costing.base_currency = pyo.units.USD_2020
```

Comment:
Is this what we use on the conventional BSM2? I think we should avoid 2020 as the base currency because of the pandemic's impact.

Reply:
Conventional also used 2020... should I switch both to 2018 or 2019?

Reply:
Good question. I'm not sure that we've selected a standard year that we want to go by. I think I can recall using 2018 for a few of the IEDO models previously. I suppose we can put a pin it this for now if we'd rather settle on a year.

Reply:
But perhaps the best route is to go with the latest year that we have the CEPCI index for

Reply:
The latest year in IDAES is 2022, but it appears that hasn't made it's way into WaterTAP yet. I tried 2021 as well, but the solver failed to return an optimal solution... not sure why that would happen.